src/cart_cell.cc

```diff
@@ -39,7 +39,7 @@ CartCell::CartCell(const Real3& position) {
 
 
   // Pointer to oxygen diffusion grid
-  ResourseManager &rm = *Simulation::GetActive()->GetResourceManager();
+  auto &rm = *Simulation::GetActive()->GetResourceManager();
```

Comment:
The recommendation is to use `auto` only when we can deduce the type when looking at the right hand side of the stmt.